config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+Doorkeeper.configure do
+  orm :active_record
+  base_controller "ActionController::Base"
+
+  # Enabled grant flows
+  grant_flows %w[client_credentials]
+
+  # Security
+  hash_token_secrets
+  authorization_code_expires_in 10.minutes
+  access_token_expires_in 2.hours
+  force_ssl_in_redirect_uri Manyfold::Application.config.force_ssl
+  forbid_redirect_uri { |uri| uri.scheme.to_s.downcase == "javascript" }
+
+  # Authentication
+  resource_owner_from_credentials { nil }
+  admin_authenticator { current_user } # Temporary
```

Comment:
You may want to limit this to only admin users, not just current user, since this UI can manage all doorkeeper applications, not just ones the user owns.

Reply:
Good catch, I meant to come back to this and check it.

Reply:
Logged new issue in #3962